docs/content/config/advanced/kubernetes.md

```diff
@@ -460,79 +432,18 @@ Then, configure both [Postfix][docs-postfix] and [Dovecot][docs-dovecot] to expe
                   readOnly: true
     ```
 
-**Downsides**
-
-- Not possible to access mailserver via inner cluster Kubernetes DNS, as PROXY protocol is required for incoming connections.
-
-## Let's Encrypt Certificates
+With this approach,
 
-[Kube-Lego][kube-lego] may be used for a role of Let's Encrypt client. It works with Kubernetes [Ingress Resources][k8s-network-ingress] and automatically issues/manages certificates/keys for exposed services via Ingresses.
```

Comment:
Did this section lose relevance?

Reply:
See my comment down below. Not sure why one would use Kube-Lego, _which is deprecated btw_, instead of certManager.